contractcourt/channel_arbitrator.go

```diff
@@ -132,6 +134,36 @@ type ChannelArbitratorConfig struct {
 	ChainArbitratorConfig
 }
 
+// ContractReport provides a summary of a commitment tx output.
+type ContractReport struct {
+	// Outpoint is the final output that will be swept back to the wallet.
+	Outpoint wire.OutPoint
+
+	// Incoming indicates whether the htlc was incoming to this channel.
+	Incoming bool
```

Comment:
Maybe we should make this a enum instead of bool? That way we could expose more about what type of output.

Reply:
This is also how it is reported over rpc. I'd like to postpone this until we have a good reason to change it.